src/map/clif.c

```diff
@@ -8907,11 +8908,11 @@ static void clif_charnameack(int fd, struct block_list *bl)
 			if (p == NULL && g == NULL)
 				break;
 
-			if (p != NULL) {
+			if (p != NULL && ssd->fakename_option != 1) {
```

Comment:
this will remove party name from all players by default.
probably better separate fakename command and hide party/guild?

if you want use all in one command, then here need check also fake_name for empty string